parsl/providers/slurm/slurm.py

```diff
@@ -193,13 +202,15 @@ def _status(self):
                                                          stderr_path=self.resources[job_id]['job_stderr_path'])
             jobs_missing.remove(job_id)
 
+        # TODO: This code can probably be depricated with the use of sacct
```

Comment:
I guess then if this code path is being hit, something is going wrong, rather than this being a normal/expected code path? in which case the log line right below might be better upgraded to a `warning` from a `debug`? I feel like the provide should be doing *something* to handle the missing jobs case rather than removing this path entirely - even if it's not expected any more, because I've seen enough stuff happen in the world of schedulers with unexpected output/lack of output...

Reply:
Yeah it makes sense to leave it to catch any slurm weirdness for missing jobs. I've updated the message to log as a `warning` now.